activemodel/test/cases/type/decimal_test.rb

```diff
@@ -60,6 +60,7 @@ def test_changed?
         assert type.changed?(5.0, 5.0, "5.0wibble")
```

Comment:
Now I come to think this test is not appropriate.
`model.int_column = "5.0wibble"` and `"5.0wibble".to_i` both result in `5`.

We may not need `\z` in regex.